cmd/lncli/commands.go

```diff
@@ -321,15 +326,11 @@ func listUnspent(ctx *cli.Context) error {
 			return nil
 		}
 		args = args.Tail()
-	default:
-		// No maxconfs was specified; we use max as flag;
-		// the default for ctx.Int64 (0) is *not* appropriate here.
-		maxConfirms = math.MaxInt32
 	}
 
-	if minConfirms < 0 || maxConfirms < minConfirms {
-		return fmt.Errorf("maximum confirmations must be greater or " +
-			"equal to minimum confirmations")
+	unconfirmedOnly = ctx.Bool("unconfirmed_only")
```

Comment:
force `minConfirms` to `0` when `unconfirmedOnly`?

Reply:
Maybe even give an error when `minConfirms` or `maxConfirms` is present in combination with `unconfirmedOnly`